pkg/build/gbbuild_113.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2018 Google LLC All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// house gb.build function for go1.13 and later version with -trimpath available
+// +build go1.13
+
+package build
+
+import (
+	"bytes"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	v1 "github.com/google/go-containerregistry/pkg/v1"
+)
+
+func build(ip string, platform v1.Platform, disableOptimizations bool) (string, error) {
+	tmpDir, err := ioutil.TempDir("", "ko")
+	if err != nil {
+		return "", err
+	}
+	file := filepath.Join(tmpDir, "out")
+
+	args := make([]string, 0, 7)
+	args = append(args, "build")
+	if disableOptimizations {
+		// Disable optimizations (-N) and inlining (-l).
+		args = append(args, "-gcflags", "all=-N -l")
+	}
+	args = append(args, "-o", file)
+	args = append(args, "-trimpath")
```

Comment:
I'd try to reduce the duplication here as much as possible across versions. Maybe define a separate function (`func addVersionSpecificBuildFlags([]string) []string` or something) that just adds the right flags so that the difference is just `return append(args, "-trimpath")`. This would make it easier to see the actual difference.